stan/math/prim/arr/functor/integrate_ode_rk45.hpp

```diff
@@ -44,6 +44,13 @@ namespace math {
  * method</a> as implemented in Boost's <code>
  * boost::numeric::odeint::runge_kutta_dopri5</code> integrator.
  *
+ * During ODE integration the global autodiff tape is continuously
```

Comment:
I don't understand what you're trying to say here. Can you write that in plain words and maybe I can help?

It really seems like we should warn that this function cannot be used in parallel. Meaning that nothing can touch the autodiff stack while this function is in use. (Do I have that condition right?)

Reply:
What is meant by "the adjoints of the parameter vector are used for Jacobian calculations"? And how does that interact with concurrency?